webrtc.html

```diff
@@ -4551,11 +4551,12 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             </dd>
             <dt><code>removeTrack</code></dt>
             <dd>
-              <p>Stops sending media from <var>sender</var>. The
-              <code><a>RTCRtpSender</a></code> will still appear in
-              <code>getSenders</code>. Doing so will cause future calls to
-              <code>createOffer</code> to mark the <a>media description</a> for
-              the corresponding transceiver as <code>recvonly</code> or
+              <p>Stops sending media from <var>sender</var>, pending
+              negotiation. The <code><a>RTCRtpSender</a></code>
```

Comment:
@taylor-b Worked on cleaning this up by setting track to null.  Can you check that it is fixed?